packages/website/src/pages/rides.js

```diff
@@ -25,11 +25,11 @@ class RidesPage extends Component {
           <p>Details of weekend rides are usually posted to <a href='https://www.facebook.com/PeckhamCC'>Facebook</a> and <a href='https://twitter.com/peckhamcc'>Twitter</a> towards the end of the week.</p>
           <p>First time riders are more than welcome but if you are thinking of joining us for the first time, please get in touch first so we know to look out for you!</p>
           <h3>Social Spin</h3>
-          <p>There is a club social ride on most Saturdays throughout the year and is the ideal introduction to riding with Peckham CC.</p>
+          <p>There is a club social ride on every Saturday throughout the year and is the ideal introduction to riding with Peckham CC.</p>
           <p>It's a non-drop ride, meaning we regroup at the tops of climbs and everyone rides back together.</p>
-          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours.  At the 30km point we usually decide whether to go further, sometimes splitting the group if people have to be back early.</p>
+          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours. There's always multiple groups at the library, and there's generally a group of people interested in riding further and/or slightly faster.</p>
```

Comment:
`There's` is a contraction of `There is`, so this sentence reads `There is always multiple groups` which reads a bit wonkily to me.  I should probably be `There are always multiple groups`

```suggestion
          <p>The '<Link to='/routes'>short loop</Link>' is about 60km long and takes about three hours. There are always multiple groups at the library, and there's generally a group of people interested in riding further and/or slightly faster.</p>
```